www/send-manager-mfa.php

```diff
@@ -7,14 +7,19 @@
  */
 
 use Sil\SspMfa\LoggerFactory;
-use sspmod_mfa_Auth_Process_Mfa as Mfa;
+use SimpleSAML\Auth\State;
+use SimpleSAML\Configuration;
+use SimpleSAML\Error\BadRequest;
+use SimpleSAML\Utilities;
+use SimpleSAML\XHTML\Template;
+use SimpleSAML\Module\mfa\Auth\Process\Mfa;
 
 $stateId = filter_input(INPUT_GET, 'StateId');
 if (empty($stateId)) {
-    throw new SimpleSAML_Error_BadRequest('Missing required StateId query parameter.');
+    throw new BadRequest('Missing required StateId query parameter.');
 }
 
-$state = SimpleSAML_Auth_State::loadState($stateId, Mfa::STAGE_SENT_TO_MFA_PROMPT);
+$state = State::loadState($stateId, Mfa::STAGE_SENT_TO_MFA_PROMPT);
```

Comment:
This may be an issue for a different feature branch, but I would have expected a different `Mfa::STAGE_...` constant would be used for this file, since the `Mfa::STAGE_SENT_TO_MFA_PROMPT` is for when we have sent the user to the `www/prompt-for-mfa.php` file. Was this intentionally chosen because of some alignment / similarity-in-purpose between the two files?

Reply:
Intentional only in the sense that I don't accurately know when a new/different stage is required.

Reply:
Okay. I would guess it's not worth trying to incorporate into this PR, but lets chat outside of this PR about that and figure out if it merits setting up a new STAGE_ constant.